protobuf/plugin/deepcopy/test/deepcopy.pb.go

```diff
@@ -46,93 +46,93 @@ var _ = math.Inf
 const _ = proto.GoGoProtoPackageIsVersion2 // please upgrade the proto package
 
 type BasicScalar struct {
-	Field1  float64 `protobuf:"fixed64,1,opt,name=Field1,json=field1,proto3" json:"Field1,omitempty"`
-	Field2  float32 `protobuf:"fixed32,2,opt,name=Field2,json=field2,proto3" json:"Field2,omitempty"`
-	Field3  int32   `protobuf:"varint,3,opt,name=Field3,json=field3,proto3" json:"Field3,omitempty"`
-	Field4  int64   `protobuf:"varint,4,opt,name=Field4,json=field4,proto3" json:"Field4,omitempty"`
-	Field5  uint32  `protobuf:"varint,5,opt,name=Field5,json=field5,proto3" json:"Field5,omitempty"`
-	Field6  uint64  `protobuf:"varint,6,opt,name=Field6,json=field6,proto3" json:"Field6,omitempty"`
-	Field7  int32   `protobuf:"zigzag32,7,opt,name=Field7,json=field7,proto3" json:"Field7,omitempty"`
-	Field8  int64   `protobuf:"zigzag64,8,opt,name=Field8,json=field8,proto3" json:"Field8,omitempty"`
-	Field9  uint32  `protobuf:"fixed32,9,opt,name=Field9,json=field9,proto3" json:"Field9,omitempty"`
-	Field10 int32   `protobuf:"fixed32,10,opt,name=Field10,json=field10,proto3" json:"Field10,omitempty"`
-	Field11 uint64  `protobuf:"fixed64,11,opt,name=Field11,json=field11,proto3" json:"Field11,omitempty"`
-	Field12 int64   `protobuf:"fixed64,12,opt,name=Field12,json=field12,proto3" json:"Field12,omitempty"`
-	Field13 bool    `protobuf:"varint,13,opt,name=Field13,json=field13,proto3" json:"Field13,omitempty"`
-	Field14 string  `protobuf:"bytes,14,opt,name=Field14,json=field14,proto3" json:"Field14,omitempty"`
-	Field15 []byte  `protobuf:"bytes,15,opt,name=Field15,json=field15,proto3" json:"Field15,omitempty"`
+	Field1  float64 `protobuf:"fixed64,1,opt,name=Field1,proto3" json:"Field1,omitempty"`
```

Comment:
Oh - our CI is super old (3.0.2 vs 3.2.0). I'll revert these and update CI's protoc in a follow up